libs/ui-lib/lib/ocm/components/featureSupportLevels/FeatureSupportLevelProvider.tsx

```diff
@@ -98,20 +49,16 @@ export const FeatureSupportLevelProvider: React.FC<SupportLevelProviderProps> =
     pullSecret,
     cluster?.openshiftVersion,
   );
-  const fetcher = () => FeatureSupportLevelsAPI.list().then((res) => res.data);
-  const { data: featureSupportLevels, error } = useSWR<FeatureSupportLevels, unknown>(
-    FeatureSupportLevelsAPI.makeBaseURI(),
-    fetcher,
-    { errorRetryCount: 0, revalidateOnFocus: false },
-  );
-  const isLoading = (!featureSupportLevels && !error) || loadingOCPVersions || isInfraEnvLoading;
+  const featureSupportLevels = useSupportLevelsAPI('features', openshiftVersion, cpuArchitecture);
 
-  const supportLevelData = React.useMemo<FeatureSupportLevelsMap>(() => {
-    if (!featureSupportLevels || error) {
-      return {};
+  const supportLevelData = React.useMemo<NewFeatureSupportLevelMap>(() => {
+    if (!featureSupportLevels) {
```

Comment:
This memo doesn't seem useful anymore, as it's basically like having twice the amount of data. 
We could either provide the default value `{}` in the `useSupportLevelsAPI` hook, or by evaluating it later
```
  const featureSupportLevels = useSupportLevelsAPI('features', openshiftVersion, cpuArchitecture);
```

Reply:
It's implemented in the same way that in old feature support levels code. Why do we need to change it?